vars/nodeWithLabels.groovy

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/groovy
+
+def call(labelSet, body) {
+    labels = ((labelSet as Set) - [null]).join('&&')
+    print "Requesting node with ${labels}"
+    node(labels) {
+      body()
+    }
+}
```

Comment:
Is this necessary? Could we not just have a helper that parses the given parameters and outputs a valid labels string that we can pass to any node?